.github/workflows/codeql-codescan.yml

```
@@ -0,0 +1,44 @@
+    steps:
+       - name: Checkout repository
+         uses: actions/checkout@v4
+
+       - name: Initialize CodeQL
+         uses: github/codeql-action/init@v2
+         with:
+          languages: ${{ matrix.language }}
+          debug: true
+
+       - name: Autobuild
+         uses: github/codeql-action/autobuild@v2
```

actionlint (1.7.4)

the runner of "github/codeql-action/autobuild@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)